### ISSUE TITLE
Needs updating?

### ISSUE THREAD

**Issue by doughnet** [directus/sdk-php#45]
Haven’t been an update for a while. Why? PHP is not dead.

**Comment by JonathanSchndr** [comment]
any infos? thanks! @benhaynes

**Comment by doughnet** [comment]
Disappointing that there's no attention to this.

**Comment by benhaynes** [comment]
I agree! Not seeing many PRs from the community, unfortunately. But the good news is...

The upcoming Directus 9 has a full OpenAPI Specification (eg: Swagger) that can be used to generate API Clients (eg: SDKs)... so there won't be a need to create these wrappers manually. We don't have time internally, and clearly the community doesn't have a desire to contribute their time to this, so automatic is better!

**Comment by JonathanSchndr** [comment]
Great news! It is possible to upgrade directus 8 to 9 or its a breakingchange?

**Comment by benhaynes** [comment]
It's a breaking change, but since Directus stores pure SQL, you can always seamlessly migrate your project content between versions... it's just the system settings which will need to be reconfigured (they are far more powerful, and there's no way to reliably convert them).
